bin/dockerfilelint

```diff
@@ -17,6 +17,9 @@ var argv = require('yargs')
     desc: 'Output linting results as JSON, equivalent to `-o json`.',
     type: 'boolean'
   })
+  .alias('v', 'version')
+  .version(function() { return require('../package').version; })
```

Comment:
I know this file gives the impression that this tool is compatible with Node versions below 4, but it's actually not. In this light, we can "simplify" (mainly for reability/brevity) this line to this:

```js
.version(() => require('../package.json').version)
```